cmd/homi/setup/cmd.go

```diff
@@ -534,12 +534,10 @@ func gen(ctx *cli.Context) error {
 			bridgeNodesJsonBytes, _ = json.MarshalIndent(enInfos[:1], "", "\t")
 		}
 		scnGenesisJsonBytes, _ := json.MarshalIndent(genIstanbulGenesis(ctx, scnAddress, nil, serviceChainId), "", "\t")
+
 		var dockerImageId string
-		if scnNum > 0 {
-			dockerImageId = "klaytn-base"
-		} else {
-			dockerImageId = ctx.String(dockerImageIdFlag.Name)
-		}
+		dockerImageId = ctx.String(dockerImageIdFlag.Name)
```

Comment:
It seems this line and declaration of dockerImageId can be merged in a single line.